flashlight/lib/text/decoder/lm/KenLM.cpp

```diff
@@ -12,7 +12,7 @@
 #ifdef USE_KENLM_FROM_LANGTECH
 #include "language_technology/jedi/lm/model.hh"
 #else
-#include <kenlm/lm/model.hh>
```

Comment:
If using C++17, you might be able to use `__has_include` to search `kenlm/lm/model.hh` first then fallback to `lm/model.hh`.

https://github.com/AnthonyCalandra/modern-cpp-features#__has_include
https://en.cppreference.com/w/cpp/preprocessor/include

Reply:
@mthrok great call -- had no idea it was in the standard now. I'll add it and we'll see what MSVC does 👍